src/components/banks/VaultTransaction/index.tsx

```diff
@@ -112,7 +113,9 @@ export const VaultTransaction: FC<Props> = ({
                   <TextInput
                     type="number"
                     value={value}
+                    min={0}
                     onChange={onChange}
+                    onKeyDown={blockInvalidChar}
```

Comment:
kudos for adding this 👏🏻

Reply:
This here stops users from typing in negative numbers or e.